### PR TITLE
Rename Pubkey Resource DAO functions

### PR DESCRIPTION
**Pull request by lzap**
It wasn't clear from the function name if I am listing pubkeys or resources or what exactly.

### CODE REVIEW

**Comment by ezr-ondrej** [comment]
Just to run tests

File: internal/dao/dao_interfaces.go [hunk 1]
```diff
@@ -30,19 +30,18 @@ type AccountDao interface {
 
 var GetPubkeyDao func(ctx context.Context) PubkeyDao
 
-// PubkeyDao TODO
+// PubkeyDao represents Pubkeys (ssh key pairs) and Resources (uploaded ssh key pairs).
```
Comment (ezr-ondrej) [inline internal/dao/dao_interfaces.go:33]:
```suggestion
// PubkeyDao represents Pubkeys (public part of ssh key pair) and corresponding Resources (uploaded pubkey to specific cloud provider in specific region).
```

**Review by ezr-ondrej — APPROVED** [review]
I'd slightly improved the comment if possible, but nothing major, I've restarted the tests, will merge tomorrow :)

**Comment by lzap** [comment]
Rebased.

**Comment by ezr-ondrej** [comment]
Thanks @lzap ! :tada:

**Pull request merged**